service/src/lib.rs

```diff
@@ -227,6 +227,7 @@ mod logger;
 pub use logger::Logger;
 
 pub use shuttle_common::database;
+pub use shuttle_common::storage_manager::StorageManager;
```

Comment:
Can we move this down to runtime too?

Reply:
I assume you mean move the re-export, which yes, we can :smile: The storage manager itself is used in a lot of crates that I'm not sure yet if will depend on shuttle-runtime. Maybe we can revisit this when we know.